homeassistant/components/bitcoin/sensor.py

```diff
@@ -120,48 +96,49 @@ def update(self):
-            self._state = f"{stats.trade_volume_usd:.1f}"
+            self._attr_state = f"{stats.trade_volume_usd:.1f}"
         elif self.type == "difficulty":
-            self._state = f"{stats.difficulty:.0f}"
+            self._attr_state = f"{stats.difficulty:.0f}"
         elif self.type == "minutes_between_blocks":
-            self._state = f"{stats.minutes_between_blocks:.2f}"
+            self._attr_state = f"{stats.minutes_between_blocks:.2f}"
         elif self.type == "number_of_transactions":
-            self._state = str(stats.number_of_transactions)
+            self._attr_state = str(stats.number_of_transactions)
         elif self.type == "hash_rate":
-            self._state = f"{stats.hash_rate * 0.000001:.1f}"
+            self._attr_state = f"{stats.hash_rate * 0.000001:.1f}"
         elif self.type == "timestamp":
-            self._state = stats.timestamp
+            self._attr_state = stats.timestamp
         elif self.type == "mined_blocks":
-            self._state = str(stats.mined_blocks)
+            self._attr_state = str(stats.mined_blocks)
         elif self.type == "blocks_size":
-            self._state = f"{stats.blocks_size:.1f}"
+            self._attr_state = f"{stats.blocks_size:.1f}"
         elif self.type == "total_fees_btc":
-            self._state = f"{stats.total_fees_btc * 0.00000001:.2f}"
+            self._attr_state = f"{stats.total_fees_btc * 0.00000001:.2f}"
         elif self.type == "total_btc_sent":
-            self._state = f"{stats.total_btc_sent * 0.00000001:.2f}"
+            self._attr_state = f"{stats.total_btc_sent * 0.00000001:.2f}"
         elif self.type == "estimated_btc_sent":
-            self._state = f"{stats.estimated_btc_sent * 0.00000001:.2f}"
+            self._attr_state = f"{stats.estimated_btc_sent * 0.00000001:.2f}"
         elif self.type == "total_btc":
-            self._state = f"{stats.total_btc * 0.00000001:.2f}"
+            self._attr_state = f"{stats.total_btc * 0.00000001:.2f}"
         elif self.type == "total_blocks":
-            self._state = f"{stats.total_blocks:.0f}"
+            self._attr_state = f"{stats.total_blocks:.0f}"
         elif self.type == "next_retarget":
-            self._state = f"{stats.next_retarget:.2f}"
+            self._attr_state = f"{stats.next_retarget:.2f}"
         elif self.type == "estimated_transaction_volume_usd":
-            self._state = f"{stats.estimated_transaction_volume_usd:.2f}"
+            self._attr_state = f"{stats.estimated_transaction_volume_usd:.2f}"
         elif self.type == "miners_revenue_btc":
-            self._state = f"{stats.miners_revenue_btc * 0.00000001:.1f}"
+            self._attr_state = f"{stats.miners_revenue_btc * 0.00000001:.1f}"
         elif self.type == "market_price_usd":
-            self._state = f"{stats.market_price_usd:.2f}"
+            self._attr_state = f"{stats.market_price_usd:.2f}"
+        self._attr_extra_state_attributes = {ATTR_ATTRIBUTION: ATTRIBUTION}
```

Comment:
This doesn't need to update every time the entity updates.
Please move this line into the constructor.